examples/deckgl/src/index.css

```
@@ -1,9 +1,14 @@
 @import 'tailwindcss';
 
+@keyframes shimmer {
+  0% { transform: translateX(-100%); }
+  100% { transform: translateX(200%); }
+}
+
 @import '@sqlrooms/ui/tailwind-preset.css';
```

Stylelint (17.9.0)

[error] Invalid position for `@import` rule (no-invalid-position-at-import-rule)

(no-invalid-position-at-import-rule)